### PR TITLE
Add perl-minion-backend-sqlite

### PR DESCRIPTION
**Pull request by khaled196**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/perl-minion-backend-sqlite/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set name = "perl-minion-backend-sqlite" %}
+{% set version = "v5.0.7" %}
```
Comment (bgruening) [inline recipes/perl-minion-backend-sqlite/meta.yaml:2]: the version string should not start with a v character, as this propagates to the real version of the package.

File: recipes/perl-minion-backend-sqlite/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+{% set sha256 = "cdde3d22b1affa7df6b1e12b2a52e9f3c1b681c8b593a1be4de3be68e4da5c72" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/D/DB/DBOOK/Minion-Backend-SQLite-v5.0.7.tar.gz
```
Comment (bgruening) [inline recipes/perl-minion-backend-sqlite/meta.yaml:10]: You should also use the name and the version variable here

File: recipes/perl-minion-backend-sqlite/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - perl-mojolicious
+    - perl-dbi
+    - perl-dbd-sqlite
+    - make
+
+  run:
+    - perl =>5.32
+    - perl-dbi
+    - perl-dbd-sqlite
+    #- perl-mojolicious
```
Comment (bgruening) [inline recipes/perl-minion-backend-sqlite/meta.yaml:35]: why is this commented?

File: recipes/perl-minion-backend-sqlite/meta.yaml (hunk 1 above)
Comment (khaled196) [reply recipes/perl-minion-backend-sqlite/meta.yaml]: @bgruening 

Expected version 5.0.7 but found v5.0.7 at /mnt/sdc/khaled/miniconda3/conda-bld/perl-minion-backend-sqlite_1689685276573/test_tmp/run_test.pl line 8.

**Review by bgruening — APPROVED** [review]

**Pull request merged**